### ISSUE TITLE
[Feature Request] Allow requests to specify tags for Sonarr/Radarr

### ISSUE THREAD

**Issue by GreyAsteroid** [tidusjar/Ombi#2969]
I have 2 users in my household that watch anime, one prefers subs and the other prefers dubs. 
As it stands they can request media and they will be auto approved and downloaded but there is no way for them to specify whether or not they want the media subbed or dubbed so they kind of just need to hope for the best.

The solution to this problem is giving the user the option to specify whether they want the media subbed or dubbed. Sonarr and Radarr do not have ways to actually choose between the two unless you setup tags and attach them to the media they apply.

So getting to the point, I'd like users to be able to choose tags (in this case "subbed" and "dubbed") when they request media and have it applied to the media requested in Sonarr/Radarr.

**Comment by Yapsonark** [comment]
What about dual audio, it’s been a while since I downloaded them because I only watch with subs but I used to get anime’s with dual audio as well as subs. One anime with the option for both

Just gotta set it up in restrictions

**Comment by GreyAsteroid** [comment]
Dual audio + subs would be ideal but a lot of media is not available in that format especially recently released anime episodes. 
I have a set of restrictions for both subbed and dubbed media applied to their respective tags. As it stands though I have to apply tags manually and potentially re-download a copy in the format that the user is actually wanting if it didn't get it right by coincidence the first time. 
I've made it possible for my users to create issues flagging media as needing to be subbed or dubbed but again that's something that only works after the fact with me intervening as apposed to an automated process where the user doesn't need me to potentially "fix" the release for them.

**Comment by tidusjar** [comment]
I'm not sure on what we would call this in the UI. My thought process would be allow the Admin to specify the tags to send over to Radarr/Sonarr. And then somehow let the user set the tags before Requesting (since it could be auto approve, so it needs to be done before/during requesting).

Maybe a dropdown stating "Request with Options" and then they have the available tags from what the admin set...

**Comment by tedrock** [comment]
I personally would like to just see in the Sonarr and Radarr setting pages an option to set a tag. Simply so when I look at an item in Sonarr or Radarr I can tell where it came from. I current have all my RSS lists applying their own tags for the same reason. This feature request, though similar, might need it’s own post, let me know if that’s the case.

**Comment by tidusjar** [comment]
A default tag that is applied to everything? That's also easily doable

**Comment by ceptonit** [comment]
> A default tag that is applied to everything? That's also easily doable

Not sure what you mean exactly but a tag that the admin could assign to users to be passed on to Sonarr/Radarr would be nice.
As tedrock mentioned, it would allow admins to know who requested what when browsing Sonarr/Radarr and potentially also apply PP scripts in there based on tags.

**Comment by webvictim** [comment]
+1 for this request. It'd be awesome to be able to set tags for a given Ombi user which will be passed through to Sonarr/Radarr.

**Comment by frazzyfin** [comment]
I would appreciate this feature so that Radarr/Sonarr can tell that something has come from Ombi, that way I can get Radarr to send a Telegram notification to the film requester that their requested film has been grabbed/downloaded etc.

**Comment by goldenpipes** [comment]
Create new user management permissions.

☑️ User prefers anime with subtitled original audio.
☑️ User prefers anime with Dubbed audio.
(only allow one checkbox to be checked)

Sonarr/Radarr settings, add a field for tags dropdown for with subs, and dubs user settings.

Anime is a hot mess...

**Comment by stevezau** [comment]
+1 for this request, will allow me to prioritise indexer type. .

**Comment by timartinelli** [comment]
+1 for this request, will allow me to prioritise indexer type. .

**Comment by MacGyver27** [comment]
Any progress on this? I have sister spamming my Movie library and wanted to easily filter her requests.. thx

**Comment by hawkinspaul** [comment]
Same, this feature would be a godsend.

**Comment by tidusjar** [comment]
No progress yet, but I'm compiling a list of things to get done soonish, this is one of them

**Comment by hawkinspaul** [comment]
That's great to hear, thanks!

**Comment by shift-faced** [comment]
> 
> 
> I personally would like to just see in the Sonarr and Radarr setting pages an option to set a tag. Simply so when I look at an item in Sonarr or Radarr I can tell where it came from. I current have all my RSS lists applying their own tags for the same reason. This feature request, though similar, might need it’s own post, let me know if that’s the case.

I like this implementation the most. I'd like _all_ Anime content that's been requested via Ombi to be automatically tagged in Sonarr, users' preferences be damned 😆

**Comment by living7ikelarry** [comment]
> No progress yet, but I'm compiling a list of things to get done soonish, this is one of them

That's good to know. I use tags to restrict Sonarr downloads to specific release group. This feature would be huge. Thanks

**Comment by Smalfinn** [comment]
I would love to see this feature too!

**Comment by Wmartin417** [comment]
This would be a great addition. I would at the very least, like to be able to set anime to be tagged as anime, since I have series tagged as anime to use different release profiles and scripts than standard series.

**Comment by PwnPeter** [comment]
I am also interested, this would be a very interesting feature !

**Comment by tidusjar** [comment]
It’s 100% on the roadmap

**Comment by Popupkiller** [comment]
Any updates on this?
I thought Ombi was tagging stuff in Radarr already, but then a new request came in without a tag. Turns out I had manually tagged them at one point, and forgot about it.

**Comment by constaac** [comment]
This would be incredible for me

**Comment by Smexhy** [comment]
I would love to see this happen!

**Comment by MacGyver27** [comment]
The guy is useless.. he promised it 2 years ago and did nothing..

**Comment by webvictim** [comment]
> The guy is useless.. he promised it 2 years ago and did nothing..

Don’t be rude. People do this stuff for free, and you could always write it yourself.

**Comment by MacGyver27** [comment]
> Don’t be rude. People do this stuff for free, and you could always write it yourself.

You might call it rude, I call it truth..

**Comment by tidusjar** [comment]
@MacGyver27 ombi is open source. Add the feature yourself if you need it so bad.

**Comment by sephrat** [comment]
In these past few months, we've added radarr tags and sonarr tags to Ombi. In the settings page you'll be able to:

- Add the requested user as a tag
- Set default tags

Please open a new issue or reach us on Discord for support.
